### PR TITLE
exercise_1.12.2

### PR DESCRIPTION
**Pull request by RuthAtem**
On this exercise ,i experimented different operators on expressions and recorded the changes

### CODE REVIEW

File: Ruth/Chapter_1/exercise_1.12.1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This is my exercise
```
Comment (ch3nkula) [inline Ruth/Chapter_1/exercise_1.12.1]: @RuthAtem, what is happening here? Why is the content of this file different? Is this exercise_1.12.1? I doubt, have a look at it!

File: Ruth/Chapter_1/exercise_1.12.2 [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Python 2.7.12 (default, Dec  4 2017, 14:50:18)
```
Comment (ch3nkula) [inline Ruth/Chapter_1/exercise_1.12.2]: This look great, so fix the issue with exercise_1.12.1. Thanks!

**Comment by RuthAtem** [comment]
Ok  thanks let me do it

**Comment by RuthAtem** [comment]
Please did I succeed in fixing it?

**Comment by ch3nkula** [comment]
I think this looks okay! I'll look at it again!

**Comment by ch3nkula** [comment]
You can move to the next exercise!

**Pull request merged**